doc/galgebra.md

```diff
@@ -0,0 +1,2009 @@
+| windows | Go to http://miktex.org/download (other downloads). Download a net installer. Install a full version of MikTex. |
+| mac | Go to http://www.tug.org/mactex/ and follow instructions to install MacTeX. |
+
+“Ipython notebook” Options
+--------------------------
+
+To use *ipython notebook* with `galgebra` it must be installed. To install *ipython notebook* do the following.
+
+Google “get-pip.py” and click on the first entry “get-pip.py”. Then follow the instructions to download “get-pip.py”. Open a terminal/command line in the directory of the download and execute `python get-pip.py` for windows or `sudo python get-pip.py` for linux. The reason for install *pip* in this manner is that it insures the correct settings for the version of python you are using. Then run in a terminal/command line `pip install ipython[notebook]`. If you have already installed *ipython notebook* you should enter `pip install ipython[notebook] –upgrade` to make sure you have the latest version. Linux and OSX users will have to use `sudo` with the commands. The version of *ipython notebook* we are using is **jupyter** and that should be shown when the notebook is started.
+
+Note that to correctly print latex from *ipython notebook* one must use the `Format()` function from the *printer* module. Go to the section on latex printing for more information.
+
+The ANSI Console
+----------------
+
+The `printer` module of `galgebra` contains the class `Eprint` which is described in section ([stdprint]). This function uses the capabilities of the ansi console (terminal) for enhanced multivector printing where multivector bases, sympy functions and derivatives are printed in different colors. The ansi console is native to Linux and OSX (which is really Unix under the hood), but not windows. The best available free substitute for the ansi console on windows is *ConEmu*. The web page for ConEmu is <http://conemu.github.io/>. In order to install *ConEmu* download the appropriate version of the *ConEmu* installer (exe file) for your system (32 bit or 64 bit) from the website and and execute it. Instructions for using *ConEmu* are given in section ([stdprint]).
+
+Geany Programmers Editor
+------------------------
+
+*Geany* is a very nice *free* programmers editor that work well with *python*. From within *geany* you can execute a *python* program. The *galgebra* printing system is setup so that you can display the program output on an ansi terminal or if you are using the LaTeXoptions has the terminal launch a *pdf* browser to view the LaTeXoutput. To install *geany* on Linux use the command line `sudo apt-get install geany`, on Windows go to <http://www.geany.org/Download/Releases> or to install *geany* in OSX go to <http://wiki.geany.org/howtos/osx/running>.
+
+What is Geometric Algebra?
+==========================
+
+<script type="text/x-mathjax-config">
+MathJax.Hub.Config({TeX: { equationNumbers: { autoNumber: "AMS" } }});
+</script>
```

Comment:
I need to enable equation auto numbering for the references to work. This script will only not be executed if I feed this .md directly to sphinx.